action.yml

```diff
@@ -79,6 +79,9 @@ inputs:
   number_of_assignees:
     description: 'Number of people to assign a PR to'
     required: false
+    contains_all_labels:
+    description: 'A comma seperated list of labels used to check if all of them are present on a pull request'
+    required: false
```

Comment:
We should be able to use the existing `labels` input here.

Reply:
i kept it separate since it gives more option in future. e.g if we want to add checks like "one of", "none of", etc.

Reply:
Actually, I just realized this can be accomplished within github workflows already

Reply:
https://docs.github.com/en/actions/learn-github-actions/expressions#contains